### DISCUSSION TITLE
Using Velero to clone

### DISCUSSION THREAD

**Discussion by kontaras** [vmware-tanzu/velero#6656] [Community support Q&A]
A common backup/restore use case is to backup a deployment and immediately restore it to create an exact replica in the same cluster. Looking at the velero docs, there is support to remap namespaces to clone from one namespace into another. Is there any support to do other remappings to allow cloning within the same namespace?

**Comment by Satsank** [comment]
Not unless you delete it before restoring...

**Reply by kontaras** [reply]
...which defeats the purpose of cloning.
